### PR TITLE
Gherkindoc Re-imagined

### PR DESCRIPTION
**Pull request by jayden-chan**
This PR includes a full rewrite of the Gherkindoc XLSX generation code using `xlsx-populate` instead of `exceljs` as the backend. There are also a few other improvements:

* Scenarios include a `pending` designation by default that can be updated by the testers as the scenarios are tested.
* Example table template variables (`<variableExample>`) are highlighted in purple

Unfortunately `xlsx-populate` still has some problems with internal hyperlinks... When viewed locally everything works fine, but when imported into the browser the link text changes to the destination address instead of the label text (ie. `Program Analytics` turns into `PROGRAMANALYTICS!A1`). 

The links still work though which means this can now be used to generate QA matrices. It's just ugly

### CODE REVIEW

File: packages/gherkindoc/test/FormatSupport.feature [hunk 1]
```diff
@@ -0,0 +1,39 @@
+Feature: Formatting support
```
Comment (loganvolkers) [inline packages/gherkindoc/test/FormatSupport.feature]: @jayden-chan I tried to write these as real self-describing behaviours. You could add some more here, as a form of documentation

**Comment by jayden-chan** [comment]
@loganvolkers DocString support is added, I've also updated it to accept `Rule` blocks, although adding those to the feature file seemed to upset the parser...

File: packages/gherkindoc/package.json [hunk 2]
```diff
@@ -2,10 +2,17 @@
   "name": "@saasquatch/gherkindoc",
```
Comment (loganvolkers) [inline packages/gherkindoc/package.json]: This NPM name is what we're going to use.

```suggestion
  "name": "picklesdoc",
```

**Comment by jayden-chan** [comment]
Moved to another repo

**Pull request closed without merging**